src/Components/Operator.js

```diff
@@ -32,32 +36,26 @@ const Calculator = () => {
     return digits;
   };
 
-  const calculate = () => {
-    setCalc(eval(calc).toString());
-  };
-
-  const clearAllData = () => {
-    setCalc('');
-  };
-
   return (
     <div>
       <div className='calculator'>
-        <div className='display'>{calc || 0}</div>
+        <div className='display'>
+          {result ? <span>({result})</span> : ''}
+          {calc || 0}
+        </div>
         <div className='operators'>
           <button onClick={clearAllData}>AC</button>
           <button>+/-</button>
```

Comment:
- [ ] <b>REQUIRED:</b> Obviously, the above operator can not function since it has no click event attached to it. It wasn't mentioned in the previous review, but I strongly recommend you implement its function before you merge your PR.

Reply:
Okay sure

Reply:
Okay sure